media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -23,7 +23,10 @@
 body {
     margin: 0;
     overflow: hidden;
-    font-family: 'Fira Sans Light', sans-serif;
+}
+
+* {
```

Comment:
D'oh, I guess this did need to be universal after all. Sorry for that.

Reply:
no worries, I had forgotten